ui/app/AppLayouts/Onboarding/stores/BIP39_en.qml

```diff
@@ -0,0 +1,19 @@
+import QtQuick 2.13
+
+ListModel {
```

Comment:
@alexandraB99 I am always for keeping such things on the backend rather (either having them hardcoded as it's done [here](https://github.com/status-im/status-desktop/blob/master/src/app_service/common/signing_phrases.nim) or load from the file on the backend) and expose that as list/model to qml. We should avoid mixing domain/logic and UI things.

Reply:
@saledjenic yes, we have discussed this with Rabbits and concluded to put the model in QML